src/vs/workbench/browser/parts/editor/breadcrumbsControl.ts

```diff
@@ -168,6 +169,7 @@ export class BreadcrumbsControl {
 		@IThemeService private readonly _themeService: IThemeService,
 		@IQuickOpenService private readonly _quickOpenService: IQuickOpenService,
 		@IConfigurationService private readonly _configurationService: IConfigurationService,
+		@ITextResourceConfigurationService private readonly _textResourceConfigurationService: ITextResourceConfigurationService,
```

Comment:
Don't have two configuration services (see below)

Reply:
Left a reply over there as well which is relevant here.